### DISCUSSION TITLE
Rename repo to Heimdall

### DISCUSSION THREAD

**Discussion by rwp0** [Perl-Oshun/oshun#15] [General]
Any plans for renaming, perhaps in the future?

As with Ovid/Cor.

I think `data-checks` is too generic and at least is something that I didn't remember searching.

**Comment by Ovid** [comment]
Might do. I was thinking about that myself. I think I'll open the repo to the public after the Toronto conference. Should do it before then.

**Comment by Ovid** [comment]
This is now done. Thanks for the suggestion!

**Reply by rwp0** [reply]
Great, I'm happy to see that:

![image](https://github.com/Ovid/heimdall/assets/68062695/0144ed1f-5526-4b26-b858-a1a1dc44b877)

Thanks
